app/controllers/concerns/idv_step_concern.rb

```diff
@@ -14,9 +15,7 @@ module IdvStepConcern
     before_action :confirm_personal_key_acknowledged_if_needed
     before_action :confirm_idv_needed
     before_action :confirm_letter_recently_enqueued
-    before_action :confirm_no_pending_gpo_profile
-    before_action :confirm_no_pending_in_person_enrollment
```

Comment:
Hey @jmhooper, for future PRs that affect in_person_enrollments, could you please tag @identity-joy-engineers for a review? It would help us track potential changes to our flow.
 
If you’re concerned about turnaround time, you can also post the PR in the #login-team-joy-eng channel and tag the @login-joy-engineers group. Thanks!